### PR TITLE
53 amplitude indicator for each channel

### PR DESCRIPTION
**Pull request by leandertolksdorf**
https://user-images.githubusercontent.com/55246283/202860413-478e1d49-934c-4b4b-93bf-458d8a9c8b22.mov

### CODE REVIEW

**Review by motschel123 — APPROVED** [review]
Works on my machine. 
Not sure about the need of the bouncing balls, but surely looks very cute! 👍 
(Maybe constantly display to highest/lowest voltage values so far)

**Review by leon-juenemann — APPROVED** [review]
Looks good. Fancy bouncing balls :)

File: README.md [hunk 1]
```diff
@@ -4,6 +4,15 @@
...
 
+# About The Project
+This project - SOSCI is a software developed based on an oscilloscopes and its features.
+
+### Product Vision
+The objective of SOSCI is to enable users of oscilloscope a way to gain access to all the features and possibilities through a software that can be done through a hardware oscilloscope and beyond. Our vision is to create a method that would better everyday life of engineers. 
+
+### Product Mission
+To organize features of an oscilloscope and processing of information and make it possible for the user to access it through a webpage. To continuously raise the level of experience of the customer with use of current technologies and maximize the output of our customer. Using SOSCI application for displaying sensor data on the frontend. 
+
```
Comment (jenswaechtler) [inline README.md:15]: why is this and the other deliverables in this pull request?

File: Apps/frontend/src/views/SineWave.svelte [hunk 2]
```diff
@@ -1,59 +1,71 @@
...
   let numberOfChannels = 10;
-  let channel_samples = Array.from(Array(numberOfChannels), () => new Array(CANVAS_WIDTH).fill(0.0))
-  let lines = []
+  let channel_samples = Array.from(Array(numberOfChannels), () =>
+    new Array(CANVAS_WIDTH).fill(0.0)
+  );
+  let lines = [];
 
-  let x = 0
+  let x = 0;
```
Comment (jenswaechtler) [inline Apps/frontend/src/views/SineWave.svelte:21]: we need a linter :roll_eyes:

**Review by jenswaechtler — CHANGES_REQUESTED** [review]
Looks good to me. 
Would it be possible to freeze the voltage value where the line indicator is located?
image

**Comment by leandertolksdorf** [comment]
> Looks good to me. Would it be possible to freeze the voltage value where the line indicator is located? image

I see your point, but there's also a min value. So I think it would make more sense displaying the current, the min and max values for each channel. What do you think?

File: Apps/frontend/src/views/SineWave.svelte (hunk 2 above)
Comment (leandertolksdorf) [reply Apps/frontend/src/views/SineWave.svelte:21]: we do.

File: README.md (hunk 1 above)
Comment (leandertolksdorf) [reply README.md:15]: hmmm. i think i created the branch from main by accident...

**Comment by jenswaechtler** [comment]
The indicator is a matter of interpretation. The current status probably fulfills the issue. If you want, you can still include the maximum value.

**Review by jenswaechtler — APPROVED** [review]

**Pull request merged**